src/Octoshift/Services/OctoLogger.cs

```diff
@@ -88,6 +94,11 @@ private string MaskSecrets(string msg)
                 .Replace(Uri.EscapeDataString(secret), "***");
         }
 
+        foreach (var redactionPattern in _redactionPatterns)
+        {
+            result = redactionPattern.Replace(result, "***");
```

Comment:
if you keep a list of strings, this line will change to the following which uses the static `Replace` method instead of allocating new Regex objects or using many pre-allocated ones.
```suggestion
            result = Regex.Replace(result, redactionPattern, "***");
```

Reply:
Addressed in 342be5b